src/host/ncp_host.cpp

```diff
@@ -148,7 +139,6 @@ void NcpHost::Init(void)
 void NcpHost::Deinit(void)
 {
     mNcpSpinel.Deinit();
-    mNetif.Deinit();
```

Comment:
Do we need to call `Deinit()` somewhere else?

Reply:
Updated! Added it in `Application::DeinitNcpMode`.